utam-core/src/main/java/utam/core/element/BasicElement.java

```diff
@@ -60,6 +60,13 @@ public interface BasicElement extends UtamBase {
    */
   String getValue();
 
+  /**
+   * Returns the string value of the CSS property with the given name
+   * @param propertyName the name of the CSS property whose value to return
+   * @return the value of the CSS property
+   */
+  String getCssPropertyValue(String propertyName);
```

Comment:
pls apply format - there will be new line before param

Reply:
Done